### ISSUE TITLE
CW 2.4: Report a Fault submissions should no include AdminArea ID in the AdminAreaDescr field

### ISSUE THREAD

**Issue by tonypesdm** [tonypesdm/CAMS#110]
Report a Fault submissions currently have the AdminAreaDescr populated with both Name and ID as supplied by the mapserver AdminArea template (in the form [Admin Area]|[ID] ).  This data should have been split and only the name saved to the AdminAreaDescr.

At the moment reported faults that have not been processed appear in the My Reported Faults in the form: Abergavenny|123.

**Comment by joerichardsESDM** [comment]
Yes, this is split into two separate fields in the new submissions
